.github/workflows/integrate.yaml

```diff
@@ -143,9 +142,7 @@ jobs:
         with:
           provider: microk8s
           channel: 1.24/stable
-          # Pinned to 2.9.34 due to https://bugs.launchpad.net/juju/+bug/1992833
-          # This should be fixed on release of juju 2.9.36 + libjuju 3.0.3
-          bootstrap-options: --agent-version="2.9.34"
+          bootstrap-options: --agent-version="2.9.42"
```

Comment:
```suggestion
```

Reply:
Hey @beliaev-maksim I know it should've been removed, but that would've installed pythonlib-juju 3 which we don't support in our tests. Please refer to [this](https://github.com/canonical/istio-operators/pull/177) other PR where we tried to unpin the version. In this PR I am experimenting with the latest stable version.

Reply:
@DnPlas ohh, I see
can you please add the comment then?

Reply:
Done